certs/anti-tank/hat.md

```diff
@@ -0,0 +1,53 @@
+# Heavy Anti-Tank
+
+## Role Description
+
+A HAT rifleman is a specialized operator class, and is not considered a standard operator.
+
+The primary role of the HAT rifleman is to eliminate large and heavily armored vehicle threats, such as Main Battle Tanks, and APC's or other heavy vehicles with reactive armor.
+
+### Specialized Equipment
+
+Completing the certification for HAT Rifleman will unlock the ability to operate HAT weapons. The standard HAT weapon is the Titan MPRL Compact.
+
+## Usage
+
+### Titan MPRL Compact
+
+The Titan MRPL Compact (also referred to as just a "Titan") is our solution to reliably countering hostile heavy armor, such as Main Battle Tanks, and other vehicles that have been equipped with reactive armor.
+
+The Titan is equipped with a Infrared sight that is used to lock the rocket onto the target, and once the rocket is fired, the rocket will guide itself utilizing its own infrared seeker. Due to the IR seeker in the rocket, the Titan is a Fire and Forget weapon system, requiring no further input from the operator once the rocket is fired.
```

Comment:
```suggestion
The Titan is equipped with an infrared sight that is used to lock the rocket onto the target, and once the rocket is fired it will guide itself utilizing its own infrared seeker. Due to the IR seeker in the rocket, the Titan is a Fire and Forget weapon system, requiring no further input from the operator once the rocket is fired.
```